### DISCUSSION TITLE
视频默认720p不知道是什么原因

### DISCUSSION THREAD

**Discussion by huanghuanhuahuh** [the1812/Bilibili-Evolved#3898] [综合讨论 / General]
无论是否安装脚本都是
想问问大家有没有这种情况
顺便请求一个解决办法, 每次都要手动切1080p很痛苦...

**Comment by Geeyun-JY3** [comment]
因为 b 站会记忆分辨率

**Reply by huanghuanhuahuh** [reply]
是的, 以前是这样, 最近不知道为什么只记忆720p了... 就算这次改成1080下次打开还是720  (T_T)

**Reply by timongh** [reply]
没啊，我这边设置了1080，下次打开还是1080。会不会是你最近改过浏览器设置，在关闭时自动清 Cookie。

**Comment by liorth** [comment]
你不是一个人。我猜就是想节省带宽，就来恶心非大会员用户。

**Comment by dot-super** [comment]
我之前也是这样，刚才我在看这个视频的时候，看到播放器默认画质是自动(720p)，我改成1080p后，发现其他视频都变成1080p，不用再手动切了，你们试试看有用没
视频地址：https://www.bilibili.com/festival/2021bnj?bvid=BV1fr4y1K7aR&p=1&t=0

**Reply by liorth** [reply]
似乎有用，这是什么原理哦。。。

**Reply by xdwangwei** [reply]
好像有用，我刚才也一直720p，点进这个视频换1080p后，再点开其他的就自动1080p了

**Reply by nbhy** [reply]
确实可以

**Reply by King-Nameless** [reply]
这好神奇_(:зゝ∠)_

**Reply by qsxc369** [reply]
有用，太玄学了吧

**Reply by taka8rie** [reply]
顶不住，真的有用。

**Reply by luckylele666** [reply]
好想知道原理啊

**Reply by Asteroiddust** [reply]
我测，真有用，测试时间2023年10月20日01:03:29

**Reply by Slotheve** [reply]
玄学有用, 测试时间2023年10月20日01:03:29

**Reply by Arvinllnamellused** [reply]
还真行

**Reply by EdwardKwok24** [reply]
特地登陆上来评论，你真是神！

**Comment by Ychan98** [comment]
我也这样，假如问题一直存在的话能否脚本添加一个功能播放视频时默认最大清晰度

**Comment by taka8rie** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
